include/libtorrent/aux_/export.hpp

```diff
@@ -69,9 +69,12 @@ POSSIBILITY OF SUCH DAMAGE.
 #elif TORRENT_USE_CRYPTOAPI
 # define TORRENT_CRYPTO_NAMESPACE inline namespace capi {
 # define TORRENT_CRYPTO_NAMESPACE_END }
-#elif defined TORRENT_USE_LIBCRYPTO
+#elif defined TORRENT_USE_LIBCRYPTO && !defined TORRENT_USE_WOLFSSL
```

Comment:
could you avoid this `&& !`-condition by moving the `USE_WOLFSSL` check ahead of it?

Reply:
yes